web/content/version-2.3/index.md

```diff
@@ -1,5 +1,6 @@
 ---
```

Comment:
This feels like a weird design. I would expect something like:

* https://docs.theforeman.org/index.html a generated list of all available version. No link directly to any docs. The source for this is in the `gh-pages` branch.
* https://docs.theforeman.org/nightly/index.html -> comes from `master` branch and lists the documents in nightly
* https://docs.theforeman.org/2.3/index.html -> comes from `2.3` branch and lists the documents in 2.3

Another possible scheme:

* master branch is just the site for the index. This also determines which versions are linked
* nightly branch is nightly docs
* x.y branch is for x.y docs
* gh-pages is generated from a combination of the above branches

Reply:
Problem is, directories with content generated via our AsciiDoc can't be shared with the static website. But I could create `index.html` welcome page and then `nightly.html` and `2.3.html` pages with the branch landing pages. That would work just fine.

The scheme you suggest is not bad, I'd like to keep this PR simple but it's something I will keep in mind. It's not a big change and it should bring more clarity to the deployment proces while it should not burden docs writers too much - they will probably only work in the `nightly` branch. Migration path should be pretty easy.

Reply:
Uh now I understand why I have chosen the clunky `version-2.3/` naming scheme - I could not get hugo to build a content file in the root folder: https://gohugo.io/content-management/organization/

Reply:
Damn I really don't know how to do this with hugo, I was hoping to use some simple and fast tool but hugo seems to be quite complex. I need to ask: https://discourse.gohugo.io/t/how-to-get-content-my-file-md-rendered/31582

Reply:
So found the solution, I can enable ugly links and have `/2.3.html`. Let's do this?

Reply:
I don't mind!